javascript.karma/release/karma/karma-netbeans-reporter/package.json

```diff
@@ -6,15 +6,15 @@
     "main": "index.js",
     "author": {
         "name": "Tomas Mysik",
-        "email": "tomas.mysik@oracle.com"
+        "email": "tmysik@apache.org"
     },
     "dependencies": {
     },
     "peerDependencies": {
         "karma": ">=0.10"
     },
-    "license": "CDDL-GPL-2-CP",
+    "license": "Apache License, Version 2.0",
```

Comment:
According to the documentation: https://docs.npmjs.com/files/package.json

This is an SPDX identifier, for ALv2 this would be [List](https://spdx.org/licenses/) :  Apache-2.0

Reply:
Fixed. It should be OK now.